src/edu/csus/ecs/pc2/ui/ResultsComparePane.java

```diff
@@ -0,0 +1,461 @@
+        } catch (Exception e) {
+            StaticLog.getLog().log(Level.WARNING, "Problem updating settings?", e);
+            e.printStackTrace(); // TODO 760 how to report problem to user ?
+        }
+
+    }
+    
+
+    protected void selectAndUpdatePrimaryCCSResultsDirectory(JTextField textField) {
+
+        String initDirectory = textField.getText();
+
+        try {
+            File newFile = FileUtilities.selectDirectory(initDirectory, textField, exportDirectoryLabel, "Select primary CCS results directory");
+            updateUserSetting(ClientSettings.PRIMARY_CCS_RESULTS_DIR, newFile.getCanonicalFile().toString());
+            textField.setText(newFile.getAbsolutePath());
+            textField.setToolTipText(newFile.getAbsolutePath());
+
+        } catch (Exception e) {
+            getLog().log(Level.WARNING, "Exception trying to select or update primary CCS dir" + e.getMessage(), e);
+            showMessage(this, "Problem with primary results dir", "Problem selecting or updating primary CCS results dir " + e.getMessage());
+        }
+
+    }
+
+    protected void selectUpdateExportDirectory(JTextField textField) {
+
+        String initDirectory = textField.getText();
+
+        try {
+            File newFile = FileUtilities.selectDirectory(initDirectory, textField, exportDirectoryLabel, "Select pc2 results directory");
+            updateUserSetting(ClientSettings.PC2_RESULTS_DIR, newFile.getCanonicalFile().toString());
+
+        } catch (Exception e) {
+            getLog().log(Level.WARNING, "Exception trying to select or pc2 results dir" + e.getMessage(), e);
+            showMessage(this, "Problem with primary results dir", "Problem selecting or updating pc2 results dir " + e.getMessage());
+        }
+
+    }
+
+    /**
+     * Create/Export pc2 results files
+     * 
+     * @return true if files created, else false.
+     */
+    protected boolean exportpc2ResultsFiles() {
+        
+        String pc2ResultsDirectory = pc2ResultsDirectoryTextField.getText();
+        
+        File pc2ResultsDir = new File(pc2ResultsDirectory); 
+        
+        if (! pc2ResultsDir.isDirectory()) {
+            
+            int result = FrameUtilities.yesNoCancelDialog(this, "pc2 results directory does not exist, create it?", "Create results directory");
+            if (result == JOptionPane.YES_OPTION) {
+                
+                ExecuteUtilities.ensureDirectory(pc2ResultsDirectory);
+                
+                pc2ResultsDir = new File(pc2ResultsDirectory);
+                if (! pc2ResultsDir.isDirectory()) {
+                    
+                    showMessage(this, "Cannot create directory", "Cannot create results dir: "+pc2ResultsDirectory);
+                    return false;
+                }
+            }  else {
+                return false;
+            }
+        }
+        
+        String targetResulsTSFFile = pc2ResultsDirectory + File.separator + ResultsFile.RESULTS_FILENAME;
```

Comment:
This is a pretty bizarre variable name...

Reply:
fixed typo

Reply:
I grant that the new name, `targetResulsTSVFilename`, is better... but even better would have been `targetResultsTSVFilename`  (i.e. with the word "Results" correctly spelled.  Minor point.

Reply:
focusing on other comments/bugs